Assets/Scripts/Controllers/CharacterController-DESKTOP-7IQNCQ4.cs

```diff
@@ -7,13 +7,16 @@ public class CharacterController : MonoBehaviour
 {
     private Rigidbody2D _rb;
     private Vector2 _movement;
```

Comment:
Vectors can be easily set to a default value:

`private Vector2 _movement;` -> `private Vector2 _movement = Vector2.zero; // (0f, 0f)`